CMakeLists.txt

```diff
@@ -649,8 +652,11 @@ if (EMSCRIPTEN)
     -s EXPORTED_FUNCTIONS=\"@${EMSCRIPTEN_EXPORTED_JSON}\"
```

Comment:
the `--memory-init-file 0` a few lines above this might be `-s SINGLE_FILE` which will do the same, but also ensure a single file even if we change to wasm later (or other things which might add a new file). but not sure that's the intention here or not.

Reply:
I think so, yeah. I'll switch to using SINGLE_FILE here too.